plugins/crust-gather.yaml

```
@@ -0,0 +1,51 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: crust-gather
+spec:
+  version: v0.5.3
+  homepage: https://github.com/crust-gather/crust-gather
+  platforms:
+  - bin: kubectl-crust-gather.exe
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_windows_amd64.tar.gz
+    sha256: 6c3e3697ad33ad190fd2398776fc9a2b28288147c57991878c9d48ed38063f23
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+  - bin: kubectl-crust-gather
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_linux_arm64.tar.gz
+    sha256: e4c01ab0b26af5771aba07b29ab303976841aace58b71605d816267f377d6df6
+    selector:
+      matchLabels:
+        os: linux
+        arch: arm64
+  - bin: kubectl-crust-gather
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_linux_amd64.tar.gz
+    sha256: 61b8dc87d4cabccc628fce55359b30d14b2d2363667d17696b98ba1f825f8582
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - bin: kubectl-crust-gather
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_darwin_arm64.tar.gz
+    sha256: 506fb5be53170cb2f9edb0842b616aed8e76c8b832f61d9e46ec2f636d880500
+    selector:
+      matchLabels:
+        os: darwin
+        arch: arm64
+  - bin: kubectl-crust-gather
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_darwin_amd64.tar.gz
+    sha256: 506fb5be53170cb2f9edb0842b616aed8e76c8b832f61d9e46ec2f636d880500
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  shortDescription: Plugin to collect cluster state and serve it via an api server
```

Sorry for pestering you but everything is a plugin in Krew so it's obvious :) 
```suggestion
  shortDescription: Collect cluster state and serve it via an api server
```

---

Np, I’ve updated the description to give more details on that front, and simplified the short one :)

---

Typically usage strings in description are not ideal but you can fix in a follow up.